API.js

```diff
@@ -77,7 +77,7 @@ export const useGoalsPull = () => {
       }
     }
   `;
-  const { loading, error, data, refetch } = useQuery(GoalsQuery, { pollInterval: 500 });
```

Comment:
Now that review, this poll interval we need to find another solution to ensure the page is refreshed

Reply:
Never mind - i see that you found a navigation event listener solution